tests/profiles/path_test.py

```diff
@@ -0,0 +1,66 @@
+import io
+
+from fontTools.ttLib import TTFont
+
+from fontbakery.checkrunner import WARN
+from fontbakery.codetesting import (assert_results_contain,
+                                    CheckTester,
+                                    TEST_FILE,
+                                    assert_PASS)
+from fontbakery.profiles import path as path_profile
+
```

Comment:
this file is missing a code-test for `com.google.fonts/check/path_short_segments`